### DISCUSSION TITLE
Reputation Loss due to Civ or Opfor driving vehicles into walls

### DISCUSSION THREAD

**Discussion by OverlordZorn** [Vdauphin/HeartsAndMinds#1408] [Question and answer] [answered]
Hi, 

on our adaptation of Hearts and Minds, we noticed that we get a lot of random Reputation damage out of no where, mainly because some AI, be it civilians or Opfor, driving their vehicles into walls, buildings, fences etc and we, "the side that has nothing to do with this" are getting blamed.

Is there a way to refine the system to only produce reputation damage when the playerside is actually accountable for?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
It has been fixed here https://github.com/Vdauphin/HeartsAndMinds/milestone/79?closed=1
Cheers
